media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -7,7 +7,7 @@ inherit meson optfeature udev
 
 if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="https://gitlab.freedesktop.org/${PN}/${PN}.git"
-#	EGIT_BRANCH="master"
+	EGIT_BRANCH="master"
```

Comment:
Is `EGIT_BRANCH` needed at all?

Reply:
You get a warning otherwise, and if someone was using the old ebuild, that one set the branch to some outdated one. I'm not sure whether the new ebuild would switch over to default master automatically or keep using the old branch value with the new `GIT_REPO_URI`, so better safe than sorry?